examples/DynamicRowHeightExample.js

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
The component here is used for the "Dynamic Row Heights" example.
It renders a table with 100k rows where each row has a different row height.

Without moving internal-state out of the redux store, this example runs slow in FDT v2.